Develop with okteto (#81)
okteto/docs: README.md

This repository is the source for https://okteto.com/docs. If you'd like to contribute to the documentation, please first check out our [contributing guide](CONTRIBUTING.md) for info on how to get started.

## Development

### ☁️ Develop with okteto

> - [Install okteto cli](https://www.okteto.com/docs/getting-started/#installing-okteto-cli)

Reviewer: ```suggestion
- [Install Okteto CLI](https://www.okteto.com/docs/getting-started/#installing-okteto-cli)
```

> - [Configure okteto context for cloud](https://www.okteto.com/docs/getting-started/#configuring-okteto-cli-with-okteto-cloud)

Reviewer: ```suggestion
- [Set your `okteto context`](https://www.okteto.com/docs/getting-started/#configuring-okteto-cli-with-okteto-cloud)
```

> - Clone the repo and then run `okteto up` from the terminal

Reviewer: ```suggestion
- Clone the repo and run `okteto up` from the root of the repo
```
